README.md

```diff
@@ -70,7 +70,7 @@ let ENV = {
 ```
 
 **Note:**
-- The images like `/ember-logo.png` will need to be in your `public` folder in your addon.
+- The images like `/ember-logo.png` will need to be in your `tests/dummy/public` folder in your addon.
```

Comment:
```suggestion
- The images like `/ember-logo.png` will need to be in your `tests/dummy/public/assets` folder in your addon.
```

I always have more consistent results if I add an assets folder in public.

Reply:
Hmm, can you clarify the issues you've seen @MelSumner? I was able to put them in `public/images/` in an app project:

https://github.com/elwayman02/hella-drunk/tree/master/public/images

I assume building the demo app for an addon should work similarly; what issue led you to need `public/assets/` specifically?

Reply:
I'm going to merge this for now because it's strictly an improvement on the current documentation, but I'd still like to hear more context about the need to use `assets/`. We can make a separate PR to update this if needed once this conversation concludes.

Reply:
Sorry @elwayman02 I missed this comment entirely. I think it's fine as-is, we can clarify if other folks run into it.